cmd/deploy/command_configuration.go

```diff
@@ -93,10 +93,18 @@ func setDeployOptionsValuesFromManifest(ctx context.Context, deployOptions *Opti
 func getStackServicesToDeploy(ctx context.Context, composeSectionInfo *model.ComposeSectionInfo, c kubernetes.Interface) ([]string, error) {
 	svcs := []string{}
 
-	for _, composeInfo := range composeSectionInfo.ComposesInfo {
-		svcs = append(svcs, composeInfo.ServicesToDeploy...)
-	}
 	if len(composeSectionInfo.ComposesInfo) > 0 {
```

Comment:
why dont we loop `composeSectionInfo.ComposesInfo` ?

Reply:
Because if it has the section [with services](https://www.okteto.com/docs/reference/okteto-manifest/#deploy-with-compose) we need to only deploy those, but if it's empty we need to deploy all (previous used by args)